package.json

```diff
@@ -13,10 +13,10 @@
   "dependencies": {
     "@bufbuild/connect-web": "^0.6.0",
     "@grpc/grpc-js": "^1.7.3",
-    "@openfeature/js-sdk": "~1.0.0",
```

Comment:
This doesn't really make sense here.

Having it seems to be no problem, since the package.jsons all include this is a peer dep with whatever version they need, but I wanted to take it out of the prod deps.